action/protocol/staking/candidate_statemanager.go

```diff
@@ -19,6 +19,7 @@ type (
 	// CandidateStateManager is candidate manager on top of StateMangaer
 	CandidateStateManager interface {
 		protocol.StateManager
+		CandCenter() CandidateCenter
```

Comment:
CandidateStateManager  shouldn't inherit protocol.StateManager

Reply:
right, this is your earlier comment, already captured in https://github.com/iotexproject/iotex-core/issues/2087 couple of days ago

it will be the next step after finish this PR